.circleci/config.yml

```diff
@@ -711,9 +711,12 @@ workflows:
       # Note that we need additional jobs for the cronjob snapshot tests because there
       # is no easy way to detect whether a job runs inside of a cronjob or specific
       # workflow. See: https://circleci.com/ideas/?idea=CCI-I-295
-      - snapshot_tests_local_browsers
-      - view_engine_snapshot_test_cronjob
-      - mdc_snapshot_test_cronjob
+      - snapshot_tests_local_browsers:
+        filters: *only_main_branch_filter
```

Comment:
Disclaimer: I don't actually know if this will work as expected, I'm basing it on how we've dealt with similar situations in other places. As far as I can tell, there's no good way of testing it without merging in the changes.

Reply:
I tried parsing the config and it looks like this needs to be indented as above with other jobs using filters.

Reply:
Thanks, updated 👍